config/install/README.md

```diff
@@ -1,101 +1,591 @@
-# Install Kuadrant and Sail via OLM
+# Install and Configure Kuadrant and Sail via OLM using the kubectl CLI
+
+This document will walk you through setting up the required configuration to install kaudrant using [kustomize](https://kustomize.io/) or a tool that leverages kustomize such as kubectl along with OLM. It will also go through more advanced configuration options to enable building up a resilient configuration. You can view the full configuration built here: [Full AWS Example](https://github.com/Kuadrant/kuadrant-operator/tree/main/config/install/full-example-aws).
+
+
+
+1. [Basic Install](#basic-installation)
+
+2. [Configure DNS and TLS integration](#configure-dns-and-tls-integration)
+
+3. [External Redis for Rate Limit Counters](#use-an-external-redis)
+
+4. [Limitador Resilient Configuration](#limitador-topologyconstraints-poddisruptionbudget-and-resource-limits)
+
+5. [Authorino Resilient Configuration](#authorino-topologyconstraints-poddisruptionbudget-and-resource-limits)
+
+4. [[OpenShift Specific] Setup Observability ](#set-up-observability-openshift-only)
+
 
 ## Prerequisites  
-- Clone the[ Kuadrant-operator](https://github.com/Kuadrant/kuadrant-operator) repo
-- OLM (operator lifecycle manager)
-- cert-manager 
+- OCP or K8s cluster and CLI available.
+- OLM installed [operator lifecycle manager releases](https://github.com/operator-framework/operator-lifecycle-manager/releases)
+- (Optional) Gateway Provider Installed: By default this guide will install the [Sail Operator](https://github.com/istio-ecosystem/sail-operator) that will configure and install an Istio installation. Kuadrant is intended to work with [Istio](https://istio.io) or [Envoy Gateway](https://gateway.envoyproxy.io/) as a gateway provider before you can make use of Kuadrant one of these providers should be installed.  
+- (Optional) cert-manager:
   - [cert-manager Operator for Red Hat OpenShift](https://docs.openshift.com/container-platform/4.16/security/cert_manager_operator/cert-manager-operator-install.html)
   - [installing cert-manager via OperatorHub](https://cert-manager.io/docs/installation/operator-lifecycle-manager/)
-- AWS, Azure or GCP with DNS capabilities. (Optional)
-- Accessible Redis instance, for persistent storage for your rate limit counters. (Optional)
+- (Optional) Access to AWS, Azure or GCP with DNS services. 
```

Comment:
For this guide, it's just AWS that's involved. (see comment below about marking that step as optional).
I'd remove mention of Azure and GCP here, or alternatively add info about how you'd use them instead of AWS in the relevant sections below.

Reply:
I will add a link to the examples for other providers